resources/js/policies/RoomPolicy.js

```diff
@@ -10,6 +10,10 @@ export default {
    */
   create (permissionService) {
     return permissionService.currentUser && permissionService.currentUser.permissions.includes('rooms.create');
+  },
+
+  delete (permissionService, model) {
+    return model.isOwner || permissionService.currentUser.permissions.includes('rooms.delete');
```

Comment:
Shouldn't the permission be created in the seeder?

Reply:
Good point